index.html

```diff
@@ -37,6 +37,128 @@ <h1 class="name">
 			<div class="bottom_geometry"></div>
 		</section>
 
+		<section class="project_section">
+			<h1 class="projects_label">
+				Projects
+			</h1>
+			<div class="project_card">
+				<div class="project_image"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
```

Comment:
- [ ] The list of project technologies should be wrapped by the `ul` tag and each item wrapped by the `li` tag. **Please apply it to other projects too.**
- [ ] Languages are not centered as an original design. Please fix it.  
**Apply it to other projects too.**
![image](https://user-images.githubusercontent.com/38635158/148404426-40097237-c9fc-4bce-a3ac-3c334aa3ac4c.png)

### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/148404574-f70c4ab2-5cbc-4a87-b6f5-46ecb39499ab.png)